src/main/java/fi/csc/avaa/smear/dto/TimeSeries.java

```diff
@@ -1,18 +1,23 @@
 package fi.csc.avaa.smear.dto;
 
+import lombok.AccessLevel;
 import lombok.Builder;
 import lombok.Getter;
 
+import javax.json.bind.annotation.JsonbDateFormat;
 import java.time.LocalDateTime;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
+
+import static fi.csc.avaa.smear.dto.DateTimeFormat.ISO8601_DATETIME_WITH_MILLIS;
```

Comment:
This could possibly be wrapped as a settings property in the future. Not show-stopper